test/uk/gov/hmrc/nisp/utils/BreadcrumbSpec.scala

```diff
@@ -47,6 +50,18 @@ class BreadcrumbSpec extends UnitSpec with OneAppPerSuite {
       MockBreadcrumb.generateHeaderUrl()(fakeRequestSP, nispUser) should not include "Gaps+in+your+record"
     }
 
+    "return an item text without a 'name' variable, when user.name has no value" in {
```

Comment:
I think should be `"return the breadcrumb..."` or `"return a URL..."` as the "item text" refers to the array of key value pairs in the query string.